### ISSUE TITLE
Show crafting history

### ISSUE THREAD

**Issue by Gan-N** [CyclopsMC/IntegratedTerminals#50]
#### Issue type:

- :heavy_plus_sign: Feature request

____

#### Short description:

1. Display the history of a synthetic article in the crafting job terminal

2. In the crafting job terminal, you can lock the quantity of a synthetic history record, and keep the locked items in the network no less than the set value

**Comment by rubensworks** [comment]
1. What do you mean by _history of a synthetic article_?
2. What do you mean by a _synthetic history record_?

**Comment by Gan-N** [comment]
1. In my imagination, it will display the finished work on the interface like the history record of Google browser.
![image](https://user-images.githubusercontent.com/35490649/76163830-8ef1a580-6184-11ea-8833-f22ec3a788b8.png)
2. In crafting job terminal, I can directly perform a historical task here. It's like visiting Google's browser history again

**Comment by rubensworks** [comment]
Ah, like that. I've updated the title.

So basically, what you want is to be able to see the crafting job history, and optionally re-trigger any of those jobs.

**Comment by Gan-N** [comment]
> Ah, like that. I've updated the title.
> 
> So basically, what you want is to be able to see the crafting job history, and optionally re-trigger any of those jobs.

Yes, my ability of expression is not very good, please forgive me
